drivers/espi/Kconfig.xec_v2

```diff
@@ -152,3 +152,17 @@ config ESPI_PERIPHERAL_ACPI_EC_IBF_EVT_DATA
 	  Enable reading event data in ACPI EC IBF ISR
 
 endif #ESPI_XEC_V2
+
+if ESPI_PERIPHERAL_8042_KBC
+
+config ESPI_PERIPHERAL_KBC_IBF_EVT_DATA
```

Comment:
Does it make sense to combine the 3 new Kconfigs into a single config?  Will all 3 configs always be enabled or disabled together?

Reply:
Depending on OS environment, these 3 new Kconfigs can be enabled/disabled separately. So we didn't combine them.

Reply:
Non blocking comment: If you don't have a specific use case that needs to enable these Kconfigs separately, I would recommend combining them.